### DISCUSSION TITLE
Duration E newscast

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#666] [Ideas]
I was spinning ideas around my head about the newscast that says you must fly a Duration E mission before you can attempt a Moon landing.  I think a straight implementation of that would be horribly unfair to someone who's chosen the minishuttle path, but maybe we could do _something_ with it.

One thought was maybe allowing the shuttles to fly Duration E for the remainder of the game, but that seems problematic.  How would we justify the shuttle's max Duration arbitrarily increasing?

Another idea was that maybe this newscast would bump up the LM Points requirement to 4 for the minishuttle player, giving them something they can do to get over this hurdle.  But how is the player supposed to know this?  And it seems hard to justify in terms of realism.  "The newscast didn't say anything about LM tests!" comes to mind.

Or maybe we could implement it but ensure the newscast doesn't come up for anyone flying shuttles - or, maybe, prevent it occurring for anyone who doesn't have anything with a capacity greater than D.  So if someone had gone from Gemini to XMS-2, they would be eligible to receive the newscast.

Or, just leave it alone.  Just throwing this one out there.

**Comment by hweimer** [comment]
Bumping the minishuttle to maximal duration E after receiving the newscast looks most reasonable to me. Not sure if you have to justify the change, but one could some explanation to the manual. For example, one could say that the additional space for supplies comes at the expense of some scientific equipment.

Currently, all event cards are chosen at the beginning of the game, i.e., way before the player has decided which programs to fly. Not sure if we want to break that.

**Reply by peyre** [reply]
That, of course, is in the event we decide to implement this newscast.  I'm fine with us leaving it nonfunctional if we prefer.

If we did what you're thinking, I can update the help file so the minishuttle description mentions that with a government mandate, they can be outfitted with extra consumables so as to achieve an E duration.  Currently Duration E (and F) missions are assigned a random number of days in the E/F range to add to the crew's records.  I then could change it to assign minishuttle Duration E to 13, the minimum number of days for that level.
